smart-contracts/contracts/BridgeProxy/ArbitrumOrbitNativeBridgeProxy.sol

```diff
@@ -55,9 +53,6 @@ contract ArbitrumOrbitNativeBridgeProxy is BridgeProxy {
         revert AssetMismatch(l2token, asset);
       }
 
-      // Take the ERC20 tokens from the sender
-      IERC20(l2token).transferFrom(sender, address(this), amount);
```

Comment:
Logic moved to the the `RelayBridge` contract because every bridge requires/implements it